pkg/infra/iac2/templates_compiler.go

```diff
@@ -174,6 +178,22 @@ func (tc TemplatesCompiler) RenderPackageJSON() (*javascript.NodePackageJson, er
 	return &mainPJson, nil
 }
 
+func validTemplateMethod(method reflect.Value) error {
```

Comment:
Part of refactor, or unrelated changes?

Reply:
This is to support eg. VpcLink's `Name()` function instead of `.Name` field.